models/user-model.js

```diff
@@ -15,7 +14,7 @@ const UserSchema = new Schema({
   outcomingRequests: [String],
   incomingRequests: [String],
   avatar: {type: String, default: 'http://tachyons.io/img/avatar_1.jpg'},
-  createdVia: { type: String},
+  oauth: { type: Boolean},
```

Comment:
why? we need to know the source.

facebook/google/apple etc

Reply:
also u missed facebook/google id

Reply:
I don't understand what facebook/google id and what information do we need from facebook again?

Reply:
Email on facebook can be changed )

So we need to find user by facebook id, it is more reliable way

Reply:
okay, got it